paasta_tools/setup_chronos_job.py

```diff
@@ -209,6 +210,23 @@ def main():
         log.warn("Skipping %s.%s: Parent job could not be found" % (service, instance))
         sys.exit(0)
 
+    # Check if the service is attempting to access un-whitelisted volumes
```

Comment:
complete_job_config is the JSON blob that we want to send to Chronos, which would contain the volumes that the service is permitted to mount. We need to load the instance config from the yaml files so we can see what the service actually wishes to mount to compare the two.